acquire/utils.py

```diff
@@ -73,6 +73,11 @@ def create_argument_parser(profiles: dict, modules: dict) -> argparse.ArgumentPa
         action="store_true",
         help="compress output (if supported by the output type)",
     )
+    parser.add_argument(
+        "--targetd",
+        action="store_true",
+        help="Setup and install targetd agent",
```

Comment:
```suggestion
        help="setup and install targetd agent",
```

Reply:
Addressed in https://github.com/fox-it/acquire/pull/105/commits/f3b924ff8ccc3f5cb866901e4c2415a2614c5b89